paper/main.tex

```diff
@@ -85,100 +92,142 @@ \section{Background}
-code. They competed against all other entries, a copy of themselves and a random
-strategy. The winner was decided on the average score a strategy achieved. The
-winner of both tournaments was the simple strategy Tit For Tat which cooperated
-on the first turn and then simply copied the previous action of it's opponent.
-Due to the strategy's strong performance in both tournaments, and moreover in a
-series of evolutionary experiments~\cite{Axelrod1981}, Tit For Tat was thought
-to be the most robust basic strategy in the IPD.
-
-However, further research proved that the strategy had weakness, and more
-specifically, it was shown that the strategy suffered in environments with
-noise~\cite{Bendor1991, Donninger1986, Molander1985, Hammerstein1984}. This was
-mainly due to the strategy's lack of generosity and contrition. The strategy was
-quick to punish a defection, and in a noisy environment it could lead to a
+strategies. Many tournaments have followed Axelrod's~\cite{Bendor1991, Stephens2002,
+Kendall2007, Stewart2012, Harper2017} and today more hundreds of strategies exist
+in the literature.
+
+Axelrod performed two widely-known computer tournaments~\cite{Axelrod1980a,
+Axelrod1980b} in the 1980's. The
+winner of both tournaments was the simple strategy Tit For Tat (TFT) which cooperated
+on the first turn and then simply copied the previous action of its opponent,
+retailiating against defections with a defection, and forgiving a defection if followed
+by a cooperation. Axelrod concluded that the strategy's robustness was due to four
+properties, which he adapted into four suggestions on doing well in an IPD:
+
+\begin{itemize}
+    \item Do not be envious by striving for a payoff larger than the opponent's payoff
+    \item Be "nice"; Do not be the first to defect
+    \item Reciprocate both cooperation and defection; Be provocable to retaliation and forgiveness
+    \item Do not be too clever by scheming to exploit the opponent
+\end{itemize}
+
+As a result of the strategy's strong performance in both tournaments, and moreover in a
+series of evolutionary experiments~\cite{Axelrod1981}, TFT was often
+claimed to be the most robust basic strategy in the IPD.
+
+There are strategies which have built upon TFT and the reciprocity based approach.
+In~\cite{Beaufils1997} Gradual was introduced which was constructed to have the
+same qualities as those of TFT except one: Gradual had a memory
+of the game since the beginning of it. Gradual recorded the number of defections
+by the opponent and punished them with a growing number of defections. It would
+then enter a calming state in which it would cooperates for two rounds. A
+strategy with the same intuition as Gradual is Adaptive Tit for
+Tat~\cite{tzafestas-2000a}. Adaptive Tit for Tat does not keep a permanent
+count of past defections, it maintains a continually updated estimate of the
+opponent's behaviour, and uses this estimate to condition its future actions.
```

Comment:
```suggestion
Tat~\cite{tzafestas-2000a}. Adaptive Tit for Tat maintains a continually updated estimate of the
opponent's behaviour, and uses this estimate to condition its future actions.
```

(Technically, the Gradual of Beaufils1997 does not keep a permanent count of past defections either.)